src/main/java/net/masterthought/cucumber/json/support/Status.java

```diff
@@ -23,7 +25,7 @@
      * @return status name as lowercase
      */
     public String getRawName() {
-        return name().toLowerCase();
+        return name().toLowerCase(Locale.US);
```

Comment:
My suggestion is to take a look at https://github.com/damianszczepanik/cucumber-reporting/tree/master/src/main/java/net/masterthought/cucumber/json/deserializers and move this change insto single class which is executed then the JSON is created instead of having the same fix in many places which are invoked many times during report creation